### PR TITLE
[#7] Extend from the official Drupal image

### PR DESCRIPTION
**Pull request by juampynr**
Instead of building our own so we can benefit
from the community's support.

### CODE REVIEW

**Comment by juampynr** [comment]
I have tested this locally using CircleCI's CLI. I am merging this and then once we get a new image at https://hub.docker.com/r/juampynr/drupal8ci/builds/ I will trigger the latest build at https://circleci.com/gh/juampynr/d8cidemo to double check that it works as expected.

**Comment by deviantintegral** [comment]
Nice!

File: .circleci/images/primary/Dockerfile [hunk 1]
```diff
@@ -1,34 +1,23 @@
...
+  mariadb-client \
+  sudo \
+  vim \
+  wget \
+  && docker-php-ext-install mysqli \
+  && docker-php-ext-install pdo \
+  && docker-php-ext-install pdo_mysql
+
+# Remove the vanilla Drupal project that comes with this image.
+RUN rm -rf /var/www/html/*
```
Comment (q0rban) [inline .circleci/images/primary/Dockerfile]: Well that was easy! Nice work. :)

File: README.md [hunk 2]
```diff
@@ -51,11 +52,14 @@ set of jobs and report their result like in the following screenshot:
...
 #### Using a custom Docker image
 
-The [.circleci/config.yml](dist/.circleci/config.yml) file uses a
-[custom Docker image](https://hub.docker.com/r/juampynr/drupal8ci/) that, although
-generic for Drupal 8 projects, may not fit yours. If this is the case, have a look at it's
+The [CircleCI configuration file](dist/.circleci/config.yml) uses a
+[custom Docker image](https://hub.docker.com/r/juampynr/drupal8ci/) that extends from
+the [official Drupal image](https://hub.docker.com/_/drupal/) and it is [hosted at
+Docker Hub](https://hub.docker.com/r/juampynr/drupal8ci/). If this image
+does not fit your project's architecture, then have a look at its
```
Comment (q0rban) [inline README.md]: Is this still accurate? It looks like it's using the official Drupal image directly.

Comment (juampynr) [reply README.md]: We are now [extending the drupal image](https://hub.docker.com/r/juampynr/drupal8ci/~/dockerfile/) by adding a few libraries on top so we can do testing. Is the text confusing? If so, can you help me to adjust it?

**Pull request merged**